okio/jvm/japicmp/build.gradle

```diff
@@ -45,6 +45,7 @@ task japicmp(type: JapicmpTask, dependsOn: 'jar') {
       'okio.SegmentedByteString', // Internal.
       'okio.SegmentPool', // Internal.
       'okio.Util', // Internal.
+      'okio.Options', // Bytecode version changed from 51.0 to 50.0    
```

Comment:
This is pretty drastic though as it means the API comparison will become useless as types are migrated since they'll all have their major version changed.

Reply:
Agree, it's not ideal. Any idea why this hasn't surfaced for other classes moved into common?